draft-ietf-rtcweb-jsep.xml

```diff
@@ -3732,11 +3736,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 MUST be returned.
                 <list style="symbols">
 
-                    <t>If the description is of type "offer", and the
-                    ICE ufrag or password changed from the previous
-                    remote description, as described in Section 9.1.1.1
-                    of <xref target="RFC5245" />, mark that an ICE
-                    restart is needed.</t>
+                    <t>If the ICE ufrag or password changed from the previous
+                    remote description, then an ICE restart is needed,
```

Comment:
The answer processing here seems kind of confusing. If the offer changed the username/password, an ICE restart is already in the works (new gathering etc), so saying that one is needed doesn't really make sense.